src/ipc-handlers.ts

```diff
@@ -99,6 +103,28 @@ export async function getInstalledApps( _event: IpcMainInvokeEvent ): Promise< I
 	};
 }
 
+export async function importSite(
+	event: IpcMainInvokeEvent,
+	{ id, backupFile }: { id: string; backupFile: BackupArchiveInfo }
+) {
+	const site = SiteServer.get( id );
+	if ( ! site ) {
+		throw new Error( 'Site not found.' );
+	}
+	const sitePath = site.details.path;
+	try {
+		const result = await importBackup( backupFile, sitePath, allValidators, allImporters );
+		if ( result?.meta?.phpVersion ) {
+			await updateSite( event, {
+				...site.details,
+				phpVersion: result.meta.phpVersion,
+			} );
+		}
+	} catch ( e ) {
+		console.log( 'Error importing site', ( e as Error ).message );
```

Comment:
When we use this IPC handler in the UI we might need to return the error to display an error message.

Reply:
This function is subject to change a little bit, as fit, when the UI is going to be implemented.
However, this is something that is going to be needed either way, so I 'll update, and return the error.